modules/mobile/app/controllers/mobile/v0/labs_and_tests_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'lighthouse/veterans_health/client'
+
+module Mobile
+  module V0
+    class LabsAndTestsController < ApplicationController
+      def index
+        response = client.list_diagnostic_reports
```

Comment:
So we're not passing any params? Would it be worth passing them to future proof his in case the front end ever needs to specify params?

Reply:
Ah shoot, meant to pass them. Thanks!